js/sign/src/integrityblock.ts

```diff
@@ -8,13 +13,28 @@ export function parseStringKey(unparsedKey: string): KeyObject {
   });
 }
 
+type SignatureAttributeKey = typeof ED25519_PK_SIGNATURE_ATTRIBUTE_NAME;
+type SignatureAttributes = { [SignatureAttributeKey: string]: Uint8Array };
+
+type IntegritySignature = {
+  signatureAttributes: SignatureAttributes;
+  signature: Uint8Array;
+};
+
+type IntegrityBlock = {
+  magic: Uint8Array;
+  version: Uint8Array;
+  signatureStack: IntegritySignature[];
+};
+
 type IntegrityBlockSignerOptions = {
   key: KeyObject;
 };
 
 export class IntegrityBlockSigner {
   private key: KeyObject;
   private webBundle: Uint8Array;
+  private integrityBlock: IntegrityBlock;
```

Comment:
`IntegrityBlock | undefined` ? (since it's not initialized in the constructor)

Reply:
Done, PTAL :)